src/main/todos.ts

```diff
@@ -59,4 +88,153 @@ export function getBasecampTodoRequest(content: string, description: string, ass
         notify: notify,
         due_on: basecampDueDate
     }
+}
+
+/**
+ * Sends requests to create new Basecamp Todos
+ * 
+ * @param basecampRequests array of BasecampTodoRequest objects to send
+ * @returns a map that associates role titles with basecamp todo ids
+ */
+export function createNewTodos(roleRequestMap: RoleRequestMap): RoleTodoIdMap {
+    const roleTodoIdMap: RoleTodoIdMap = {};
+
+    Object.keys(roleRequestMap).forEach( role => {
+        let request = roleRequestMap[role];
```

Comment:
```suggestion
        let request: BasecampTodoRequest = roleRequestMap[role];
```
Sorry also must have missed from before